packages/react-components/src/components/Dialpad/Dialpad.tsx

```diff
@@ -311,17 +302,14 @@ const DialpadContainer = (props: {
  * @beta
  */
 export const Dialpad = (props: DialpadProps): JSX.Element => {
-  /* @conditional-compile-remove(dialpad) */
-  const localeStrings = useLocale().strings.dialpad;
-  /* @conditional-compile-remove(dialpad) */
+  const localeStrings = dialpadLocaleStringsTrampoline();
   const strings = { ...localeStrings, ...props.strings };
 
-  return (
-    <>
-      {
-        /* @conditional-compile-remove(dialpad) */
-        <DialpadContainer strings={strings} {...props} />
-      }
-    </>
-  );
+  return <DialpadContainer strings={strings} {...props} />;
 };
+
+function dialpadLocaleStringsTrampoline(): DialpadStrings {
+  /* @conditional-compile-remove(dialpad) */
+  return useLocale().strings.dialpad;
+  return {} as DialpadStrings;
```

Comment:
nitty nit: Funilly `useLocale().strings.dialpad` actually exists and has the right values (only the type doesn't include those fields in the stable build).
So I would `return useLocale().strings.dialpad as unknown as DialpadStrings`

Just a nit, we don't anticipate ever using that code path